kafkamgmt/apiv1/client/docs/MetricsInstantQueryList.md

```diff
@@ -1,108 +1,112 @@
 # MetricsInstantQueryList
 
-## Properties
+    ## Properties
 
-Name | Type | Description | Notes
------------- | ------------- | ------------- | -------------
-**Kind** | Pointer to **string** |  | [optional] 
-**Id** | Pointer to **string** |  | [optional] 
-**Items** | Pointer to [**[]InstantQuery**](InstantQuery.md) |  | [optional] 
+    Name | Type | Description | Notes
+    ------------ | ------------- | ------------- | -------------
+        **Kind** | Pointer to **string** |  | [optional] 
+        **Id** | Pointer to **string** |  | [optional] 
+        **Items** | Pointer to [**[]InstantQuery**](InstantQuery.md) |  | [optional] 
+        
 
-## Methods
+    ## Methods
 
-### NewMetricsInstantQueryList
+        ### NewMetricsInstantQueryList
 
-`func NewMetricsInstantQueryList() *MetricsInstantQueryList`
+        `func NewMetricsInstantQueryList() *MetricsInstantQueryList`
 
-NewMetricsInstantQueryList instantiates a new MetricsInstantQueryList object
-This constructor will assign default values to properties that have it defined,
-and makes sure properties required by API are set, but the set of arguments
-will change when the set of required properties is changed
+        NewMetricsInstantQueryList instantiates a new MetricsInstantQueryList object
```

Comment:
I guess the template is causing all the documentation to become indented?

Reply:
yes, let me see if I can fix it.

Reply:
This is resolved now.